ganga/GangaLHCb/Lib/Applications/GaudiExec.py

```diff
@@ -195,7 +195,7 @@ class GaudiExec(IPrepareApp):
         'nMakeCores': SimpleItem(defvalue=1,
                                  doc='Number of cores to be provided via the "-j" option to the "make" command'\
                                      'when building the ganga-input-sandbox'),
-
+        'apptainerBuild' : SimpleItem(defvaule=False, doc="Run the build command in apptainer"),
```

Comment:
This name might come back and bite us. Setting this to True will get a very specific centos7 image which the name doesn't indicate. Maybe have a second option that includes the the name of the actual image.